qa/workunits/rbd/cli_generic.sh

```diff
@@ -1463,6 +1463,176 @@ test_mirror_snapshot_schedule_recovery() {
     ceph osd pool rm rbd3 rbd3 --yes-i-really-really-mean-it
 }
 
+test_mirror_group_snapshot_schedule() {
+    echo "testing mirror group snapshot schedule..."
+    remove_images
+    ceph osd pool create rbd2 8
+    rbd pool init rbd2
+    rbd namespace create rbd2/ns1
+
+    rbd mirror pool enable rbd2 image
+    rbd mirror pool enable rbd2/ns1 image
+    rbd mirror pool peer add rbd2 cluster1
+
+    test "$(ceph rbd mirror group snapshot schedule list)" = "{}"
+    ceph rbd mirror group snapshot schedule status | fgrep '"scheduled_groups": []'
+
+    expect_fail rbd mirror group snapshot schedule ls
+    test "$(rbd mirror group snapshot schedule ls -R --format json)" = "[]"
+
+    rbd create $RBD_CREATE_ARGS -s 1 rbd2/ns1/img1
+    rbd create $RBD_CREATE_ARGS -s 1 rbd2/ns1/img2
+    rbd group create rbd2/ns1/test1
+    rbd group image add rbd2/ns1/test1 rbd2/ns1/img1
+    rbd group image add rbd2/ns1/test1 rbd2/ns1/img2
+
+    test "$(rbd mirror group status rbd2/ns1/test1 |
+        grep -c mirror.primary)" = '0'
+
+    rbd mirror group enable rbd2/ns1/test1 snapshot
+
+    test "$(rbd mirror group status rbd2/ns1/test1 |
+        grep -c mirror.primary)" = '1'
+
+    rbd mirror group snapshot schedule add -p rbd2/ns1 --group test1 1m
```

Comment:
It would be good to add a separate mirror enabled image which is not part of the group, and create a  pool level `mirror snapshot schedule` and check that it does not apply to the images that are part of the group.

Reply:
I added this test to `test_mirror_snapshot_schedule()`, which tests other mirror image snapshot scheduler commands. Please see, https://github.com/ceph/ceph/pull/60627/commits/1373046df866e67edf35413cdecf8cd291068e6d